src/towncrier/test/test_builder.py

```diff
@@ -208,3 +208,13 @@ def test_ordering(self):
             - Added Fish
 """
         )
+
+
+class TestUtilityFunctions(TestCase):
```

Comment:
Very minor comment, but I think that we can have these tests in the same test case as the other tests.

------

These are unit tests, but we also need an integration/functional/end to end test to check that the default templates work.

Reply:
Done